src/MoBi.Core/Domain/Model/MoBiSpatialStructure.cs

```diff
@@ -28,7 +28,8 @@ ObjectPath getContainerPath(IContainer container)
             if (parentContainer?.ParentPath == null)
                return absolutePath;
 
-            return absolutePath.AndAddAtFront(parentContainer.ParentPath);
+            absolutePath.AddAtFront(parentContainer.ParentPath);
```

Comment:
Adds paths by appending two ```ObjectPath``` together

Reply:
this is NOT The fix. The fix is to make the two methods behave the same. One is just syntax sugar over the other